### DISCUSSION TITLE
能否在首页添加频道

### DISCUSSION THREAD

**Discussion by StrangerBWL** [the1812/Bilibili-Evolved#1880] [功能建议 / Ideas]
首页风格：清爽

如题：
首页在清爽模式中没有频道唉，每次都要点到别区， 才能点进频道，特别麻烦
![Snipaste_2021-05-23_19-58-03](https://user-images.githubusercontent.com/66122976/119259515-409dba80-bc01-11eb-821c-f410d2e6786e.png)

希望：
能把频道搞到首页来

谢谢

浏览器：Edge 90.0.818.66 (官方内部版本) (64 位)

**Comment by Geeyun-JY3** [comment]
开之前先善用搜索，重复：#1636

**Reply by StrangerBWL** [reply]
嗯，那行吧，就是有点麻烦
